ivy.el

```diff
@@ -3227,12 +3227,20 @@ Otherwise, ~/ will move home."
 
 (defvar ivy--exhibit-timer nil)
 
+(let (last-pos)
+  (defun ivy-input-changed? ()
+    (let* ((pos (line-end-position))
+           (changed (not (eq pos last-pos))))
+      (setq last-pos pos)
+      changed)))
```

Comment:
If you prefer, this can also be written as:
```el
(let ((pos (line-end-position)))
  (prog1 (not (eql pos last-pos))
    (setq last-pos pos)))
```
but it's up to you.

Reply:
I thought `let*` might be more readable, but considering the context of this small predicate function, actually wouldn't matter too much, advice taken!